packages/cli/src/commands/election/show-l2.test.ts

```diff
@@ -0,0 +1,160 @@
+      `✘  ${groupAddress} is ValidatorGroup`
+    )
+  })
+
+  it('fails when provided address is not a voter', async () => {
+    const logMock = jest.spyOn(console, 'log')
+    const [voterAddress] = await web3.eth.getAccounts()
+
+    await expect(testLocallyWithWeb3Node(Show, [voterAddress, '--voter'], web3)).rejects.toThrow(
+      "Some checks didn't pass!"
+    )
+    expect(stripAnsiCodesAndTxHashes(logMock.mock.calls[1][0])).toContain(
+      `${voterAddress} is not registered as an account. Try running account:register`
+    )
+  })
+
+  it('shows data for a group', async () => {
+    const kit = newKitFromWeb3(web3)
+    const logMock = jest.spyOn(console, 'log')
+    const [groupAddress] = await web3.eth.getAccounts()
+    const writeMock = jest.spyOn(ux.write, 'stdout')
+
+    await setupGroup(kit, groupAddress)
+
+    const getValidatorGroupVotesMock = jest.spyOn(
+      ElectionWrapper.prototype,
+      'getValidatorGroupVotes'
+    )
+    getValidatorGroupVotesMock.mockImplementation(async () => {
+      return {
+        address: '0x1000000000000000000000000000000000000001',
+        capacity: new BigNumber(3879657093998775988268146),
+        eligible: true,
+        name: 'Eligible group',
+        votes: new BigNumber(3481303474410894544646170),
+      } as ValidatorGroupVote
+    })
+
+    await testLocallyWithWeb3Node(Show, [groupAddress, '--group'], web3)
+
+    expect(writeMock.mock.calls).toMatchInlineSnapshot(`[]`)
+    expect(logMock.mock.calls.map((args) => args.map(stripAnsiCodesAndTxHashes)))
+      .toMatchInlineSnapshot(`
+      [
+        [
+          "Running Checks:",
+        ],
+        [
+          "   ✔  0x5409ED021D9299bf6814279A6A1411A7e866A631 is ValidatorGroup ",
+        ],
+        [
+          "All checks passed",
+        ],
+        [
+          "address: 0x1000000000000000000000000000000000000001
+      capacity: 3879657093998776000000000 (~3.880e+24)
+      eligible: true
+      name: Eligible group
+      votes: 3481303474410894600000000 (~3.481e+24)",
+        ],
+      ]
+    `)
+  })
+
+  it('shows data for an account', async () => {
+    const kit = newKitFromWeb3(web3)
+    const logMock = jest.spyOn(console, 'log')
+    const [voterAddress, groupAddress, anotherGroupAddress] = await web3.eth.getAccounts()
+    const writeMock = jest.spyOn(ux.write, 'stdout')
+
+    await registerAccount(kit, voterAddress)
+
+    const getVoterMock = jest.spyOn(ElectionWrapper.prototype, 'getVoter')
+    getVoterMock.mockImplementation(async () => {
+      return {
+        address: voterAddress,
+        votes: [
+          {
+            active: new BigNumber(1001),
+            group: groupAddress,
+            pending: new BigNumber(2002),
+          },
+          {
+            active: new BigNumber(3003),
+            group: anotherGroupAddress,
+            pending: new BigNumber(4004),
+          },
+        ],
+      } as Voter
+    })
```

Comment:
why did we need to mock this? it means we are not actually testing potential contract differences which was the purpose of the tests